frontend/common/utils/utils.tsx

```diff
@@ -298,6 +306,10 @@ const Utils = Object.assign({}, require('./base/_utils'), {
         valid = isScaleupOrGreater
         break
       }
+      case 'SELF_HOSTED_INVITE': {
```

Comment:
Does the permission need to be `'SELF_HOSTED_INVITE'` here? Can't it just be `'INVITE'` ?